run-tests.py

```diff
@@ -380,10 +214,10 @@ def export_notebook(ipath, opath):
     parser.add_argument(
         "--examples",
         action="store_true",
-        help="Test all Jupyter notebooks in `docs/source/examples/`.",
+        help="Test all Jupyter notebooks in `docs/source/examples/` (deprecated, use nox or pytest instead).",  # noqa: E501
     )
     parser.add_argument(
-        "-debook",
+        "--debook",
```

Comment:
Apparently, this was an error that wasn't caught. I updated the argument to match what the documentation says (or said, rather).